docs/css/fui-site.css

```diff
@@ -8843,6 +8845,39 @@ hr {
 hr {
   margin: 36px 0; }
 
+pre {
+  display: block;
+  padding: 10px;
+  margin: 0 0 20px;
+  word-break: break-all;
+  word-wrap: break-word;
+  color: #666;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 5px; }
+  pre code {
```

Comment:
Can these be merged with the block in line 8866?

Reply:
this is the compiled css file that merged docs scss and FDUI lib. it's not duplicate code.